webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -418,72 +419,79 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
     [experiment.state],
   );
 
+  const foldableRows: InfoRow[] = useMemo(() => {
+    const rows = [
+      {
+        content: experiment.description || <Typography.Text disabled>N/A</Typography.Text>,
```

Comment:
layout gets broken with long texts

<img width="974" alt="image" src="https://user-images.githubusercontent.com/109113616/228934529-5d54f43c-60e7-47d1-a4e1-36180a03a68b.png">

Reply:
This issue should be fixed. The duplicate key warnings look to be coming from the Tags component, which this PR didn't touch, so I think that's out of scope.